### ISSUE TITLE
Add Ability for Carousel to resize images

### ISSUE THREAD

**Issue by rob3000** [DoghouseMedia/Doghouse_Carousel#7]

**Comment by rob3000** [comment]
Should we have a config option to set this globally and then allow per carousel group override?

**Comment by erfanimani** [comment]
I think it should just be on a carousel group basis. Chances are the size
will be different for each group anyway and setting it globally wouldn't be
that advantageous.

It could have a select box with two options: "resize" and "no resize" for
example.

Thoughts?

**Comment by rob3000** [comment]
Yeah was thinking that, Could have a conditional field on the carousel group that will allow you to enter the resize options
